.github/workflows/robot-test.yml

```diff
@@ -174,7 +174,7 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Download Artifacts
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@6b208ae046db98c579e8a3aa621ab581ff575935 # v4.1.1
```

Comment:
this will not work, see https://github.com/centreon/centreon-collect/pull/1025

Reply:
yes indeed, this should be merged after the 1025

Reply:
there's too many conflict when I try to rebase on "MON-33526-adapt-e2e-tests-upgrade-gha"